### ISSUE TITLE
Waterfox does not work on any Discourse based website

### ISSUE THREAD

**Issue by Quisquose** [MrAlex94/Waterfox-Old#157]
Discourse (https://www.discourse.org) is an open source platform used for Forums.

A few weeks ago Discourse must have made an update because suddenly all Discourse-based forums stopped working in Waterfox at the same time (where they had all been working perfectly fine previously).

A couple of examples sites are:

**MP3Tag Forum:**  https://community.mp3tag.de
**Obsidian Forum:** https://forum.obsidian.md

In addition to the out-of-date browser warning banner, no pages can be scrolled, and sometimes certain page elements are missing entirely. Basically, discourse-based websites are completely unusable at the moment.

This was tested using:
**Browser**: Waterfox Classic v2022.08  (clean install, no profile)
**OS**: Win7 and Win10 (21H2)

**Comment by reallyuniquename** [comment]
>In addition to the out-of-date browser warning banner, no pages can be scrolled

This is purely cosmetic or maybe even a deliberately broken thing. Here is a uBlock rule that fixes scrolling.

```
community.mp3tag.de##html:style(overflow-y: unset !important;)
```

**Comment by Quisquose** [comment]
Yeah, the scroll thing is just one problem. On Obsidian, for example, the forum page is totally blank for me.

**Comment by reallyuniquename** [comment]
Obsidian works for me the same way community.mp3tag.de does.

**Comment by Squall-Leonhart** [comment]
Duplicate of https://github.com/WaterfoxCo/Waterfox-Classic/issues/150

fixed with filter ```||*.*/assets/browser-detect-*.js```
